storage_xml.c

```diff
@@ -79,13 +81,18 @@ static void handle_settings(struct xt_node *node, set_t **head)
 			}
 		}
 		set_setstr(head, name, c->text);
+		if (locked && !g_strcasecmp(locked, "true")) {
+			s = set_find(head, name);
+			if(s)
```

Comment:
style nitpick: always use `{...}`. Also, space in `if (s)`

Same thing with the `if(set->flags & SET_LOCKED)` below